pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
     "Jpype1==1.4.1",
     "pyarrow==12.0.0",
     "matplotlib~=3.6.3",
-    "pandas~=1.5.3",
+    "pandas<=1.4.3",
```

Comment:
is that strictly required for this PR ? I think this would require us to change the ODH image too.

Reply:
ODH and AIX360 Pandas versions are at odds, so one of them would have to change. ODH is pinned at `pandas~=1.5.3`, though. This change is only for this PR to allow the installation of the extras.